aptos-move/block-executor/src/code_cache_global_manager.rs

```diff
@@ -290,13 +290,20 @@ fn prefetch_aptos_framework(
     let code_storage = state_view.as_aptos_code_storage(guard.environment());
 
     // If framework code exists in storage, the transitive closure will be verified and cached.
-    let maybe_loaded = code_storage
-        .fetch_verified_module(&AccountAddress::ONE, ident_str!("transaction_validation"))
-        .map_err(|err| {
-            // There should be no errors when pre-fetching the framework, if there are, we
-            // better return an error here.
-            PanicError::CodeInvariantError(format!("Unable to fetch Aptos framework: {:?}", err))
-        })?;
+    let maybe_loaded = {
+        #[cfg(fuzzing)]
```

Comment:
nit: can we branch on cfg flag? i.e. 
```
let result = if #[cfg(fuzzing)] {
  code_storage.fetch_module(&AccountAddress::ONE, ident_str!("transaction_validation"));
} else {
  code_storage.fetch_verified_module(&AccountAddress::ONE, ident_str!("transaction_validation"));
};
```

Reply:
To branch, `cfg!(fuzzing)`, I need to make code always available. If it's okay I can remove `#cfg[fuzzing]` from `fetch_module_skip_verification`.

Reply:
Ah, it will still be evaluated at runtime? Sad...

Reply:
I think you can do this:
```
cfg_if! {
        if #[cfg(feature = "fuzzing")] {
            ...
        } else {
            ...
        }
    }
```